keyvault/azure-spring-boot-starter-keyvault-certificates/keyvault-certificates-server-side/README.md

```diff
@@ -59,7 +59,8 @@ You can also create resources manually via Azure Portal. Please follow:
    - If you cannot use managed identity, you can register your application with AAD, see [register app with AAD][register_app_with_AAD]. The registration also creates a second application object that identifies your app.
 2. Create the key vault and certificates. Please refer to [create key vault and certificates][create_key_vault_and_certificates]
 3. Make the key vault accessible to your service principal. Please refer to [assign key vault access policy][assign_key_vault_access_policy]
-4. You need manually configure the application.yml, replace the placeholders with the resources you created in the Azure Portal.
+   > Attention: The service principal must be configured with get and list permissions of certificate, key, secret.
```

Comment:
Same here.

Reply:
1. get and list are all read permission, I think it's ok to add list permission.
2. I want to be consistent with our shell
<img width="663" alt="image" src="https://user-images.githubusercontent.com/4465723/157356117-0301a2ac-b6e4-496f-a4a3-651f261e71ad.png">

Reply:
I suggest to configure minimal permissions and update the shell script.

Reply:
updated